project2/terraform/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.13.2"
```

Comment:
Should this be `~> 0.13.2` instead, in order to keep it within the 0.13 family?

Reply:
Should we disallow older versions?

I think candidates will fall into two types:
* never terraformed before in their life, but going to give it a shot
* seasoned pros who will just make it work

Reply:
I can definitely do that, I tested this against 1.X and it worked. But keeping it 0.13 would likely be best for now

Reply:
i did test this on 1.0.6, so we can lock it down to 1.x if we want.

Reply:
leaving this open for now so its more flexible for candidates and us